src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,45 @@
+const tasks = require("../task/index");
+
+module.exports.getallExpenses = async (req, res) => {
+  Task.find().then((result) => {
+    res.send({ data: result });
+  });
+};
+
+module.exports.createNewTask = (req, res) => {
+  const task = new Task(req.body);
```

Comment:
What if I don't send data? What if I don't submit the data? if I don't send all the data? or incorrect?